src/bonsai/bonsai/bim/module/drawing/svgwriter.py

```diff
@@ -828,15 +897,18 @@ def draw_text_annotation(self, text_obj: bpy.types.Object, position: Vector) ->
                         field.attrib["class"] = classes_str
 
                     if fill_bg:
-                        symbol_copied = symbol_svg.copy()
-                        for text_tag in symbol_copied.xml.findall("text"):
-                            self.add_fill_bg(text_tag, copy=False)
+                        for text_tag in symbol_svg.xml.findall("text"):
+                            text_line = text_tag.text
+                            font_size = self.get_font_size (classes_str)
+                            bg_rect = self.add_fill_bg(text_tag, text_line, font_size, copy=False)
+                            symbol_svg.add(bg_rect)
```

Comment:
@Andrej730 for some reason this doesn't work--can't figure out why.

Reply:
What doesn't work exactly?

Reply:
the `bg_rect` was not added to the svg.  Not sure why.  No errors. 
Sorry noobie. :\

Reply:
What to do you mean "not added"?
![image](https://github.com/user-attachments/assets/26f2cd0a-7d44-48e2-a253-b6b809ce5b17)